omar/openshift/python/deploy-app.py

```diff
@@ -0,0 +1,191 @@
+import argparse
+import sys
+import logging
+import json
+import os
+
+import parse_deploy_sequence
+import openshift
+import helpers
+
+logging.basicConfig(format='%(message)s', level=logging.INFO)
+# logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+
+argument_parser = argparse.ArgumentParser()
+argument_parser.add_argument('-c', '--config-file', required=True, help='The deployConfig yml file to use')
+argument_parser.add_argument('-o', '--openshift-url', required=True, help='The url of the openshift instance to deploy to')
+argument_parser.add_argument('-t', '--template-dir', default=None, required=False, help='The directory where the openshift template files are located')
+argument_parser.add_argument('-m', '--configmap-dir', default=None, required=False, help='The directory where the configmap files are located')
+argument_parser.add_argument('--all', required=False, action='store_true', help='Deploy all objects defined in the deployConfig file')
+argument_parser.add_argument('-s', '--service', required=False, help='Deploy the provided service. Overridden by the \'--all\' flag')
+argument_parser.add_argument('--remove', required=False, action='store_true', help='Remove existing objects first')
+argument_parser.add_argument('--nodeploy', required=False, action='store_true', help='Avoid deploying apps, just remove them')
+argument_parser.add_argument('--loglevel', default='INFO', type=str.upper, choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'FATAL'], help='The log level to use')
+argument_parser.add_argument('--abort-on-failure', action='store_true', help='Abort the process when there is an error')
+argument_parser.add_argument('--start-phase', type=int, default=1, help='The phase to start with, the first phase (0) by default')
```

Comment:
Change default from 0 to 1. Great catch @anaga670 !!!

Reply:
Done